### ISSUE TITLE
cortx-motr 08spiel-sns-repair-quiesce seems enter an infinite loop

### ISSUE THREAD

**Issue by yanqingfu** [Seagate/cortx-motr#4]
it runs more than three hours and still running, also it used more than 40G of the space. system info 8vCPU, 16GB RAM, 8x100GB Disks. This behavior happens randomly.

> > Filesystem                                   Size  Used Avail Use% Mounted on
> devtmpfs                                     7.8G     0  7.8G   0% /dev
> tmpfs                                        7.8G     0  7.8G   0% /dev/shm
> tmpfs                                        7.8G  1.0M  7.8G   1% /run
> tmpfs                                        7.8G     0  7.8G   0% /sys/fs/cgroup
> /dev/mapper/vg_sysvol-lv_root                 25G  3.9G   20G  17% /
> /dev/sda1                                    976M  119M  791M  14% /boot
> /dev/mapper/vg_sysvol-lv_tmp                 976M   12M  898M   2% /tmp
> /dev/mapper/vg_sysvol-lv_var                 406G   **47G**  343G  12% /var
> /dev/mapper/vg_sysvol-lv_log                 9.1G  121M  8.5G   2% /var/log
> /dev/mapper/vg_sysvol-lv_audit               240M  4.2M  219M   2% /var/log/audit
> ssc-nfs-home1.colo.seagate.com:/home/670951  1.5T  1.3T  214G  86% /home/670951
> tmpfs                                        1.6G     0  1.6G   0% /run/user/670951
> none                                         1.9T  671M  1.9T   1% /tmp/test_m0t1fs_16-08-2020_20:51:50
> tmpfs                                        1.6G     0  1.6G   0% /run/user/0

from the source code 
```
spiel_wait_for_sns_repair()
{
echo $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS
    $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS <<EOF
import time
$SPIEL_FIDS_LIST

$SPIEL_RCONF_START

one_status = SpielSnsStatus()
ppstatus = pointer(one_status)
active = 0
while (1):
    active = 0
    rc = spiel.sns_repair_status(fids['pool'], ppstatus)
    print ("sns repair status responded servers: " + str(rc))
    for i in range(0, rc):
        print "status of ", ppstatus[i].sss_fid, " is: ", ppstatus[i].sss_state
        if (ppstatus[i].sss_state == 2) :
            print "sns is still active on ", ppstatus[i].sss_fid
            active = 1
    if (active == 0):
        break;
    time.sleep(3)

$SPIEL_RCONF_STOP
EOF
}
```
if the active in the while loop is not equal to 0, it will run forever. Maybe we need to add a time limit to the code? for example, if after one hour the test was not finished, break.

08spiel-sns-repair was running ok
spiel-sns-repair: test status: SUCCESS
620.99user 316.33system 11:08.12elapsed **140%CPU** (0avgtext+0avgdata 1726428maxresident)k
23180440inputs+26713280outputs (562major+9612693minor)pagefaults 0swaps

**Comment by huanghua78** [comment]
There is some issues in DIX repair.
Tasks to enable DIX repair is in progress.

**Comment by huanghua78** [comment]
DIX repair has been fixed in e4995a5d481a6604e3fd2a26056c8b6853f91e65
